frontend/src/config/patient.schema.json

```diff
@@ -0,0 +1,145 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "patient.schema.json",
+    "title": "Patient",
+    "description": "A patient record for COVID testing",
+    "type": "object",
+    "properties": {
+      "patientID": {
+        "description": "Unique patient identifier",
+        "type": "string"
+      },
+      "patientLastName": {
+        "description": "Patient's last name",
+        "type": "string"
+      },
+      "patientFirstName": {
+        "description": "Patient's first name",
+        "type": "string"
+      },
+      "patientMiddleName": {
+        "description": "Patient's middle name",
+        "type": "string"
+      },
+      "patientSuffix": {
+        "description": "",
+        "type": "string"
+      },
+      "patientRace": {
+        "description": "Patient race",
+        "type": "string"
+      },
+      "patientDOB": {
+        "description": "Patient date of birth",
+        "type": "string",
+        "format": "date"
+      },
+      "patientGender": {
+        "description": "Patient sex",
```

Comment:
Just noting here that we should have a convo about how we want to support gender and sex in patient demographics. AFAIK the latest recommendation on this from ACLA in terms of data structure (with an eye towards backwards compatibility) is https://www.acla.com/wp-content/uploads/2018/06/2018-Jun-25-ACLA-Best-Practice-Patient-Gender-Final.pdf

Reply:
(which is what the FHIR guide references)

Reply:
Yes, agreed. For the record I built this straight from the airtable spec.